src/views/SignUpView.vue

```diff
@@ -53,7 +32,8 @@
             <the-input
             @input.capture="isCheckboxChecked = !isCheckboxChecked"
             type="checkbox"
-            idValue="privacy"
+            id="privacy"
```

Comment:
Why do you need an `id` on that input?

Reply:
Because the input itself (checkbox) has the appearence set to none and the label is styled to look like an input so I need this connection between the label and the input so as label is clicked it checks the input.

Reply:
I still don't get why you need that.
If I'd make a fake input element id place it next to the hidden one and refer to it with css.
Even if the base input has `appearance: none` it preserves it's functionality:
```css
.someInputClass:checked + .myCustomElement {
  // checked styles
}
```

`id` is needed if you use `for` attribute inside a label, however I suggest wrapping input with label when possible.
```html
<label>
  <input /><span>No need for id and for attrs now</span>
</label>

Reply:
You're right, I could have done that. 

At MMC's courses I was taught to use inputs and labels separately and apply this id and for so clicking the label would trigger the change of "state" of the input.

Recently, I've come to know I can actually place  the input inside the label but since it's "fresh" knowledge of mine, I haven't still got to use to it so much to remember it 😛

**Corrected**

Reply:
Great! (Resolve this thread anytime)